support/metrics_covalent.py

```diff
@@ -0,0 +1,652 @@
+import numpy as np
+from datetime import datetime
+
+NOW = datetime.now().date()
+
+# -------------------------------------------------------------------------- #
+#                               Helper Functions                             #
+# -------------------------------------------------------------------------- #
+def swiffer_duster(txn, feedback):
+    '''
+    Description:
+        remove 'dust' transactions (i.e., transactions with less than $0.1 in spot fiat value get classified as dust) and
+        keep only 'successful' transactions (i.e., transactions that got completed).
+
+    Parameters:
+        txn (dict): Covalent class A endpoint 'transactions_v2'
+        feedback (dict): score feedback
+
+    Returns:
+        txn (dict): formatted txn data containing only successful and non-dusty transactions
+    '''
+    try: 
+        if txn['quote_currency'] == 'USD':
```

Comment:
@irene-bbox what do you think about this?
`if txn['quote_currency'] == 'USD':`
`txn['items'] = [t for t in txn['items'] if t['successful'] and t['value_quote'] > 0]`

Reply:
Brilliant! I've implemented the list comprehension as you recommended!